### PR TITLE
Support for mapping created_at field to Hubspot

### PR DESCRIPTION
**Pull request by oneshot-engineering**

### CODE REVIEW

**Comment by oneshot-engineering** [comment]
@kpthatsme @yakkomajuri  - Can you please take a look at this quick PR? This would address my issues

File: index.js [hunk 1]
```diff
@@ -194,7 +194,12 @@ async function createHubspotContact(email, properties, authQs, additionalPropert
                     const d = new Date(eventSendTime)
                     d.setUTCHours(0, 0, 0, 0)
                     hubspotFilteredProps[hubSpotProperty] = d.getTime()
-                } else if (postHogProperty in properties) {
+                } else if(postHogProperty === 'created_at'){
+                    //TODO: This logic needs to be applied for all date type fileds.  
+                    const d = new Date(properties[postHogProperty])
+                    d.setUTCHours(0, 0, 0, 0)
+                    hubspotFilteredProps[hubSpotProperty] = d.getTime()
+                }else if (postHogProperty in properties) {
```
Comment (yakkomajuri) [inline index.js:202]:
```suggestion
                } else if (postHogProperty in properties) {
```

**Comment by oneshot-engineering** [comment]
> Looks good, just one minor suggestion

Great, thank you very much merging. Would it be deployed immediately for cloud PostHog version?

**Comment by yakkomajuri** [comment]
@oneshot-engineering I've just updated it on Cloud for everyone!

**Pull request merged**